src/SignalR/clients/java/signalr/build.gradle

```diff
@@ -22,7 +22,7 @@ allprojects {
     version project.findProperty('packageVersion') ?: "99.99.99-dev"
 
     java {
-        sourceCompatibility = 1.8
+        sourceCompatibility = 1.9
```

Comment:
Why was this changed?
> invalid source release: 9

Reply:
See https://github.com/dotnet/sdk/pull/45822#issuecomment-2579608368

We started hitting an error w/ version 8 when updating to 11.0.25:

>     EXEC : error : [options] source value 8 is obsolete and will be removed in a future release [D:\a\_work\1\vmr\src\aspnetcore\src\SignalR\clients\java\signalr\core\signalr.client.java.core.javaproj]

I'm not sure how to proceed if the underlying machines are in a mixed state, as they seem to be (some have 11.0.24, which is incompatible w/ 9, and some have 11.0.25, which is incompatible w/ 8)

Reply:
Oh, never mind, it's Helix, not the build machines. Do you know why non-windows is on JDK 10 & windows is on 11 here? The Helix failure was only on Mac.

https://github.com/dotnet/aspnetcore/blob/e0e0224d744e6219849538cf9ce5dd3db1ccd658/eng/targets/Helix.targets#L3-L4

Reply:
I thought all machines were supposed to have moved to 21.X?

Reply:
Probably should remove those lines now? Wonder if we used to have to install the jdk ourselves 5 years ago.

Reply:
Think we got our streams crossed, it's the Helix images, not the build machines - see above comment

Reply:
> Probably should remove those lines now? Wonder if we used to have to install the jdk ourselves 5 years ago.

Worth a try - the mac queue claims to have "macos-openjdk" installed, but it doesn't specify which version. Let's see what happens if we don't install our own 10.0 version

Reply:
Doesn't look like we install the jdk for mac though
> '$(IsWindowsHelixQueue)' != 'true' AND '$(IsMacHelixQueue)' != 'true'

Reply:
Oh, I totally missed that. So it looks like whatever version of `macos-openjdk` is pre-installed on the MacOS-13 queue isn't compatible w/ 1.9. @dotnet/dnceng does anyone know how we can check what version is installed on OSX.amd64.open? helix.dot.net doesn't list one.

Reply:
Related to [this ask](https://github.com/dotnet/dnceng/issues/3093)?

Reply:
https://github.com/dotnet/dnceng/issues/4554
> all macOS machines will have OpenJDK 8 and 21 after they are next refreshed

So maybe the machines haven't been refreshed?

Reply:
Now we have
> ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

On Windows 🙄